packages/vite/src/node/plugins/html.ts

```diff
@@ -367,7 +367,7 @@ export function buildHtmlPlugin(config: ResolvedConfig): Plugin {
               prop.name === 'style' &&
               prop.type === NodeTypes.ATTRIBUTE &&
               prop.value &&
-              prop.value.content.includes('url(') // only url(...) in css need to emit file
+              /url\s*\(.*?\)/.test(prop.value.content) // only url(...) in css need to emit file
```

Comment:
why did you use optional metacharacter, you could just ignore it. It do not have any effect on the output 😊

Reply:
I think `url (` must had the `)`, use the optional metacharacter break the grab.

if no optional metacharacter will grab `brackground: url (abc); brackground-image: url(efg)`

Reply:
and if no `)` is also do not have any effect on the output

Reply:
It just acts like a guard to prevent overflow, to collect strings other than ones inside the url

Reply:
test method will crap all string? 

maybe crap one will break the crap?

Reply:
may be `url\s*\(` will get the one fulfil requirements will break the crap?